LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentAuthRequestParams.kt

```diff
@@ -0,0 +1,24 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.BrowserSwitchOptions
+
+/**
+ * Local payment result information.
+ */
+class LocalPaymentAuthRequestParams internal constructor(
+    /**
+     * @return The original request used to create the local payment transaction.
```

Comment:
Can we move the doc for each property into the class doc, using the `@property` annotation in the comment? 

example: https://github.com/braintree/braintree_android/blob/474febc875cac373c3d6823b59903bb8e674dd41/PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt#L55